example-new-architecture/android/build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         compileSdkVersion = 34
         targetSdkVersion = 34
         ndkVersion = "25.1.8937393"
-        kotlinVersion = "1.8.0"
+        kotlinVersion = "1.9.0"
```

Comment:
so why doesn't it compile with Kotlin 1.8 (I know that Android SDK moved to Kotlin 1.9) - which error exactly are you getting?

Reply:
I think I can actually compile with Kotlin 1.8.21 and Gradle 7.3.3. I am running some tests on the CI with the current setup, I'll then re-run it with the new values and see if it works.

I was getting conflicting kotlin versions (duplicated kotlin-stdlib related dependencies conflict)

Reply:
Like this one? https://github.com/DataDog/dd-sdk-android/blob/fd7786e2b8cc5513fc441b76dd6dcf1c98de6d9c/MIGRATION.MD?plain=1#L36

It is better to use lower Kotlin version if possible, indeed.